CMakeLists.txt

```diff
@@ -577,6 +577,16 @@ if(BUILD_BENCHMARKS)
     if(ENABLE_REQUIRES_ALL_SYMBOLS)
         add_h3_benchmark(benchmarkPolygon src/apps/benchmarks/benchmarkPolygon.c)
     endif()
+
+    # Country benchmark: Downloads country geometry and generates the benchmark file
+    add_custom_target(generate-country-benchmark
+        COMMAND node ${CMAKE_CURRENT_SOURCE_DIR}/scripts/make_countries.js src/apps/benchmarks/benchmarkCountries.c
+        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+        )
+    
+    if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/src/apps/benchmarks/benchmarkCountries.c")
+       add_h3_benchmark(benchmarkCountries src/apps/benchmarks/benchmarkCountries.c)
+    endif()
```

Comment:
The thing I struggled with here is that I want to be able to edit the generated files and then re-run, e.g. to benchmark a single country or change the res. If I add the `generate-country-benchmark` target as a dependency, I think it'll run every time I run `make bench_benchmarkCountries`, overriding local changes. Open to ideas here.

Reply:
Perhaps this can be done with `BYPRODUCTS`? https://cmake.org/cmake/help/latest/command/add_custom_target.html I can try to look into coding that later